item_test.go

```diff
@@ -93,3 +93,32 @@ func Test_Item_ExpiresAt(t *testing.T) {
 
 	assert.Equal(t, now, item.ExpiresAt())
 }
+
+func TestItem_Version(t *testing.T) {
```

Comment:
`TestItem_Version` -> `Test_Item_Version`

Reply:
Can you also add version `assert.Equal()` checks to `Test_Item_update` and `Test_Item_touch`?